tests/test_content.py

```diff
@@ -327,7 +327,7 @@ def test_subpackage_metadata_preserved():
     err2 = ErrorBundle()
     err2.detected_type = PACKAGE_EXTENSION
 
-    submain.test_package(err1, xpi1, 'jetpack-1.8-outdated.xpi')
+    submain.test_package(err1, xpi1, 'jetpack-1.16-outdated.xpi')
```

Comment:
Given the name of the XPI, wasn't it there in the tests on purpose to test outdated XPIs?